Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -75,10 +109,51 @@
 NETWORK_NOT_FOUND = "A network was not found for this address"
 
 
-class Client(BaseClient):
-    def __init__(self, base_url, verify=True, proxy=False, ok_codes=tuple(), headers=None, auth=None, params=None):
-        super(Client, self).__init__(base_url, verify, proxy, ok_codes, headers, auth)
-        self.params = params
+class InvalidIPAddress(ValueError):
+    pass
+
+
+class InvalidNetmask(ValueError):
+    pass
+
+
+class InvalidIPRange(ValueError):
+    pass
+
+
+class IPv4AddressStatus(Enum):
```

Comment:
I thought of using an `enum` here because it has the out-of-the-box benefits such as type-safety and uniqueness of member keys:

```python
>>> class IPv4AddressStatus(Enum):
...     ACTIVE = "ACTIVE"
...     USED = "USED"
...     UNUSED = "UNUSED"
...     USED = "USED"
TypeError: Attempted to reuse key: 'USED'

>>> class IPv4AddressStatus(object):
...     ACTIVE = "ACTIVE"
...     USED = "USED"
...     UNUSED = "UNUSED"
...     USED = "USED"
```

Reply:
Leaving as `Enum`